src/Unleash/Scheduling/FetchFeatureTogglesTask.cs

```diff
@@ -58,6 +61,8 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
                 throw new UnleashException("Exception while fetching from API", ex);
             }
 
+            ready = true;
```

Comment:
This might have to get moved if we add support for the ready event

Reply:
Smells like a later problem to me